### DISCUSSION TITLE
Easy way to get a male voice?

### DISCUSSION THREAD

**Discussion by AllesMeins** [coqui-ai/TTS#1378] [General Q&A]
Hi, I’m just playing around with TTS and was wondering if there is an easy way to get a male voice. The models that come with the package seem all to be female, right? Are there any other already trained models I could download from somewhere?

I know that there are some Datasets out there, that I could use to train a model myself, but I’ve to admit that I’m way out of my depth there. I took a look at train.py and I’ve absolutely no idea how any of these options would influence the result. And from what I read I could take days on my machine for each run - so that is quite the investment for trial-and-error.

So what is the easiest way to get a male voice for somebody with a solid computer knowledge but no experience in voice synthesis?

**Comment by erogol** [comment]
Try VCTK models. They are multi speaker and have male voice.
